### DISCUSSION TITLE
Creating new account shows fewer currencies

### DISCUSSION THREAD

**Discussion by brccabral** [firefly-iii/firefly-iii#8774] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

The Welcome screen shows many currencies when creating a new account, but the Create Asset shows only Euro (version 6.1.13) and the demo shows Euro, Pound, Forint, USD, but not the full list.

### Debug information

I could not get into https://demo.firefly-iii.org/debug, but this is my local one:

Debug information generated at 2024-04-10 11:41:11 for Firefly III version **v6.1.13**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>6.1.13 / v2.0.13 / 23 (exp. 23)</td></tr><tr><td>PHP version</td><td>8.3.4 (64bits) / apache2handler / Linux x86_64</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>67108864 (64 MB)</td></tr><tr><td>Database drivers</td><td><strong>*mysql*</strong>, pgsql, sqlite, </td></tr><tr><td>Docker build</td><td><span>#</span>1052, base <span>#</span>82</td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>America/Los_Angeles + America/Los_Angeles</td></tr><tr><td>App environment</td><td>local, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>debug, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td></td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>never (never)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td></td></tr><tr><td>Session start</td><td>2024-04-01 00:00:00</td></tr><tr><td>Session end</td><td>2024-04-30 23:59:59</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_US</td></tr><tr><td>User locale</td><td>en_US</td></tr><tr><td>Locale(s) supported</td><td>en_US.utf8: :white_check_mark:<br>en_US.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:124.0) Gecko/20100101 Firefox/124.0</td></tr></tbody></table>

### Expected behaviour

When creating a new Asset Account we expect all currencies to be available for selection.

### Steps to reproduce

1. Go to Others -> Accounts -> Asset accounts
2. Open the Currency drop down
3. Only few currencies shows up (https://demo.firefly-iii.org/debug). Version 6.1.13 only shows Euro.

### Additional info

![image](https://github.com/firefly-iii/firefly-iii/assets/130455/736055a1-8eac-45ac-9919-305779b6f5c0)

![image](https://github.com/firefly-iii/firefly-iii/assets/130455/a9c73662-7ad1-4b55-908c-daca01d79a7b)

**Comment by JC5** [comment]
You can enable the currencies you would like to use under `/currencies`

**Reply by brccabral** [reply]
May I suggest adding a link to `/currencies`?

![image](https://github.com/firefly-iii/firefly-iii/assets/130455/aac4a820-1cb0-41e0-a5a1-c02fc51d7a21)
